### PR TITLE
👙 Optimize embedding layer

### PR DESCRIPTION
**Pull request by dathudeptrai**
This is a PR include small changes to optimize training speed, bellow is my observation of tacotron-2's training speed:

TF 2.2.0 (4s/1it) -> TF 2.3.0 (3.5s/1it) -> TF 2.3.0 + Faster Embedding (3.0s/1it). All brenchmarks run on 2080Ti with batch_size 32, non mixed_precision.

Related issue: https://github.com/tensorflow/tensorflow/issues/38287

### CODE REVIEW

**Comment by ZDisket** [comment]
Does it help the alignment?

**Comment by dathudeptrai** [comment]
> Does it help the alignment?

Nope :D. Faster training only. BTW, i found that after the model learn diagonal alignment, we can training with window masking, it make the aligment cleaner :))).
